test/fixtures/generator/bwx.xcodeproj/xcshareddata/xcschemes/Generator_App.xcscheme

```diff
@@ -69,9 +69,9 @@
             buildForAnalyzing = "YES">
             <BuildableReference
                BuildableIdentifier = "primary"
-               BlueprintIdentifier = "2FED647E7A6091DF616D345B"
-               BuildableName = "generator"
-               BlueprintName = "generator"
+               BlueprintIdentifier = "F5FCEC7426929BDA7F9F1875"
+               BuildableName = "tests.xctest"
+               BlueprintName = "tests"
```

Comment:
Why is the order of these changing?

Reply:
Here is what I think happened:
- I added [this sort](https://github.com/buildbuddy-io/rules_xcodeproj/blame/main/tools/generator/src/Generator/XCSchemeInfo%2BTargetInfo.swift#L180) towards the of the implementation for #803. 
- My last successful scheme generation must have occurred before I added this sort.

The order that it is being set is what I would expect it to be.

Reply:
The ordering issue appeared due to #807.